script.js

```diff
@@ -0,0 +1,746 @@
+            } else {
+                console.log("single file");
+                app.display.loopCount = 0;
+                if (app.display.allRegions) {
+                    if (app.display.regionsArray.length > 0) {
+                        region = app.display.regionsArray[0];
+                        laneData = app.state[region].laneData[app.display.loopCount];
+                        app.display.regionsArray.shift();
+                        if (laneData) {
+                            url = "bikelanes/" + laneData;
+                            app.laneAjaxQueue(url, region);
+                        } else {
+                            console.log("=== ALERT: Missing data for", region);
+                        }
+                    } else {
+                        app.display.regionsArray = ["AL", "AR", "DC", "MO", "PG"];
+                    }
+                }
+            }
+        }).fail(function(){
+            console.log("*** ajax fail T ***");
+        });
+    },
+
+    // ======= bufferAjaxQueue =======
+    bufferAjaxQueue: function(url, region, buffer) {
+        console.log("bufferAjaxQueue");
+        $.ajax({
+            url: url,
+            method: "GET",
+            dataType: "text"
+        }).done(function(jsonData){
+            console.log("*** ajax success ***");
+            var jsonData2 = $.parseJSON(jsonData);
+            var bufferFeatures = L.mapbox.featureLayer(jsonData2).addTo(app.activeMap);
+            app.state[region].bufferLayers[buffer] = bufferFeatures;
+            bufferFeatures.setStyle(app.display.bufferStyle);
+        }).fail(function(){
+            console.log("*** ajax fail T ***");
+        });
+    },
+
+    // ======= allLanesQueue =======
+    allLanesQueue: function(url, regionsArray) {
+        console.log("allLanesQueue");
+        if ((Array.isArray(regionsArray)) && (regionsArray.length > 0)) {
+            var region = regionsArray[0];
+        } else {
+            var region = regionsArray;
+        }
+
+        $.ajax({
+            url: url,
+            method: "GET",
+            dataType: "text"
+        }).done(function(jsonData){
+            console.log("*** ajax success ***");
+            var jsonData2 = $.parseJSON(jsonData);
+            var laneFeatures = L.mapbox.featureLayer(jsonData2).addTo(app.activeMap);
+            app.state[region].laneLayers.push(laneFeatures);
+            laneFeatures.setStyle(app.state[region].bikeLaneStyle);
+            if (loopCount < app.state[region].laneData.length - 1) {
+                loopCount++;
+                var laneData = app.state[region].laneData[loopCount];
+                app.allLanesQueue(url, region);
+            } else {
+                var arrayR = app.display.regionsArray;
+                arrayR.shift();
+                app.display.regionsArray = arrayR;
+                app.allLanesQueue(url, arrayR);
+            }
+        }).fail(function(){
+            console.log("*** ajax fail T ***");
```

Comment:
Need to add a user-friendly fail error in the UI instead of a developer-only console.log error eventually. This line can stay in this current PR for now.